lib/workers/repository/process/index.ts

```diff
@@ -56,13 +55,23 @@ export async function processRepo(
         baseBranchConfig.hasBaseBranches = true;
       }
       await platform.setBaseBranch(baseBranch);
-      const baseBranchRes = await extractAndUpdate(baseBranchConfig);
-      ({ res } = baseBranchRes);
+      const baseBranchRes = await extract(baseBranchConfig);
       branches = branches.concat(baseBranchRes.branches);
       branchList = branchList.concat(baseBranchRes.branchList);
     }
-    return { res, branches, branchList };
+    return { branches, branchList };
```

Comment:
This seems to be skipping `packageFiles` as expected by the calling function: `const { branches, branchList, packageFiles } = await processRepo(config);`

Reply:
packageFiles variable was ignored before my changes in this place, so that is why I made it like that
Do we need it?

Reply:
It's referenced in `const additionalFiles = await getAdditionalFiles(config, packageFiles);`

Reply:
yeah, but exactly in this place packageFiles was ignored.
Lets look in the master branch:
- https://github.com/renovatebot/renovate/blob/master/lib/workers/repository/process/index.ts#L59
as we can see packageFiles is part of baseBranchRes
and on next string packageFiles just ignored.
and further: https://github.com/renovatebot/renovate/blob/master/lib/workers/repository/process/index.ts#L64

only res, branches and branchList are returned.

That is why packageFiles is optional parameter in ExtractResult (and was optional before me).

What I'm trying to protect is that this PR is just refactoring, without changing overall logic behind it.

And additional changes going to be features or bug fixes and not related to this PR itself.